### DISCUSSION TITLE
Error while loading the doctr.io and doctr.models

### DISCUSSION THREAD

**Discussion by Arya920** [mindee/doctr#1597] [Q&A] [answered]
after installing the doctr using the code `pip install python-doctr`. I tried to load the package using the following code 
`from doctr.models import ocr_predictor

model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True)`

but getting the following error ~

OSError: cannot load library 'gobject-2.0-0': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'gobject-2.0-0'

please help 🙏😭

**Comment by felixdittrich92** [comment, marked as answer]
Hi @Arya920 :wave:,
This error is raised from the `weasyprint` installation.

with the next release you will be able to skip the weasyprint installation (extra dependency)
For the current pypi released version you can do the following:
- install weasyprint via conda: `conda install weasyprint`
Or
- try the following fix: https://github.com/Kozea/WeasyPrint/issues/1556#issuecomment-1097977671
Or
- `pip3 install "python-doctr[torch,viz]@git+https://github.com/mindee/doctr.git@main"` (this will install from the main branch)

options:
python-doctr[torch,tf,viz,html,contrib]  <--- html requires weasyprint

**Comment by Arya920** [comment]
Thank you @felixdittrich92  It solved the issue 🙏. I am closing this discussion
